src/Uno.UI.RemoteControl/HotReload/MetadataUpdater/HotReloadAgent.cs

```diff
@@ -150,17 +150,15 @@ internal UpdateHandlerActions GetMetadataUpdateHandlerActions()
 			};
 		}
 
-		MethodInfo? GetUpdateMethod(
-		[DynamicallyAccessedMembers(HotReloadHandlerLinkerFlags)]
-		Type handlerType, string name)
+		MethodInfo? GetMethod([DynamicallyAccessedMembers(HotReloadHandlerLinkerFlags)] Type handlerType, string name)
```

Comment:
Let's not diverge too much from the original code, in case we need to sync it (it's coming from dotnet watch)